modules/meshrenderinggl/src/processors/meshrasterizer.cpp

```diff
@@ -475,7 +480,9 @@ void MeshRasterization::rasterize(const ivec2& imageSize,
         for (auto mesh : enhancedMeshes_) {

```

Comment:
outside PR, but why is transferFuncUnit set here and not with the other uniforms in the process function?

Reply:
The transfer function needs to bind a texture. We cannot leave that one bound in between, but only set it right before the render call.

Reply:
Right, thought about that later as well